### PR TITLE
RHDEVDOCS:5998: Content creation for the GitOps 1.12.1 RN

### PR DESCRIPTION
**Pull request by Dhruv-Soni11**
Version(s):

1.12.1

Content from this release must be CP'ed to GitOps 1.12, 1.11, and 1.10 releases.

Issue:

https://issues.redhat.com/browse/RHDEVDOCS-5998

Link to docs preview:

https://74127--ocpdocs-pr.netlify.app/openshift-gitops/latest/release_notes/gitops-release-notes.html#release-notes-for-gitops-1-12-1_gitops-release-notes

SME review: @reginapizza 
QE review: @varshab1210 
Internal peer review: @eromanova97 
Peer review: @obrown1205 

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by Dhruv-Soni11** [comment]
Kindly note that the content for this release has been populated as per the inputs provided in in this [doc](https://docs.google.com/document/d/1U1N8fwbYrwrCv4MENsU3ReD8HWjAEBeP1EGsPQuso_k/edit). The JIRA tracker links are yet to be populated for each Fixed Issue and the content for the embargo issue is not added yet.

File: modules/gitops-release-notes-1-12-1.adoc [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+{gitops-title} 1.12.1 is now available on {OCP} 4.12, 4.13, and 4.14.
+
+[id="errata-updates-1-12-1_{context}"]
+== Errata updates
+
+[id="rhsa-XXXX-NNNN-gitops-1-12-1-security-update-advisory_{context}"]
+=== RHSA-XXXX-NNNN - {gitops-title} 1.12.1 security update advisory
+
+Issued: 2024-04-03
```
Comment (varshab1210) [inline modules/gitops-release-notes-1-12-1.adoc:18]:
```suggestion
Issued: 2024-04-08
```

File: modules/gitops-release-notes-1-12-1.adoc [hunk 2]
```diff
@@ -0,0 +1,29 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="release-notes-for-gitops-1-12-1_{context}"]
+= Release Notes for {gitops-title} 1.12.1
+
+{gitops-title} 1.12.1 is now available on {OCP} 4.12, 4.13, and 4.14.
```
[the remark this answers is not in the document]
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-12-1.adoc:10]: Fixed.

**Review by reginaelyse — APPROVED** [review]

**Comment by varshab1210** [comment]
LGTM

File: modules/gitops-release-notes-1-12-1.adoc [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+[id="release-notes-for-gitops-1-12-1_{context}"]
+= Release Notes for {gitops-title} 1.12.1
+
+{gitops-title} 1.12.1 is now available on {OCP} 4.12, 4.13, 4.14, and 4.15.
+
+[id="errata-updates-1-12-1_{context}"]
+== Errata updates
+
+[id="rhsa-2024:1753-gitops-1-12-1-security-update-advisory_{context}"]
```
Comment (adellape) [inline modules/gitops-release-notes-1-12-1.adoc:15]: Just hyphens and underscores for `id`s:
```suggestion
[id="rhsa-2024-1753-gitops-1-12-1-security-update-advisory_{context}"]
```

File: modules/gitops-release-notes-1-12-1.adoc [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+=== RHSA-2024:1753 - {gitops-title} 1.12.1 security update advisory
+
+Issued: 2024-04-08
```
Comment (adellape) [inline modules/gitops-release-notes-1-12-1.adoc:18]: I'm seeing the issue date in https://access.redhat.com/errata/RHSA-2024:1753 as 4/10:
```suggestion
Issued: 2024-04-10
```

File: modules/gitops-release-notes-1-12-1.adoc (hunk 3 above)
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-12-1.adoc:15]: Fixed.

File: modules/gitops-release-notes-1-12-1.adoc [hunk 5]
```diff
@@ -0,0 +1,29 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="release-notes-for-gitops-1-12-1_{context}"]
+= Release Notes for {gitops-title} 1.12.1
```
Comment (adellape) [inline modules/gitops-release-notes-1-12-1.adoc:8]: Mentioned on Slack about maybe a follow-up PR to get all of the section titles in this assembly using lowercase "notes" for "Release notes", as it currently is inconsistent.

**Review by adellape — APPROVED** [review]

**Pull request merged**